critiquebrainz/frontend/templates/review/modify/release_group.html

```diff
@@ -17,10 +19,5 @@
   </dl>
 </div>
 <div class="col-md-2">
-  <img id="cover-art" class="img-responsive" style="max-height:120px;max-width:120px;">
-  <script>
-    $("img").last()
-        .on("error", function () { $(this).attr("src", "/static/img/missing-art.png"); })
-        .attr("src", "//coverartarchive.org/release-group/{{ entity['id'] }}/front-250");
-  </script>
+  {{acover_art(entity['id'], 'release_group', 'id=cover-art class=img-responsive style=max-height:120px;max-width:120px;') }}
```

Comment:
@gentlecat Oops, sorry that slipped through; I'll check the final diff more thoroughly next time. Resolved in 65f3275